Assets/UI/actionpanel.lua

```diff
@@ -1079,44 +1080,52 @@ end
 
 -- ===========================================================================
 function OnLocalPlayerTurnBegin()
-    -- Standard disable is set to false in the refresh.
-    -- This extra level of input catching is done when tutorial has raised
-    -- this boolean and will prevent spam clicking through; as the tutorial
-    -- system itself sets ENABLED as it's hiding all controls.
-    if m_isSlowTurnEnable then
-        Controls.TutorialSlowTurnEnableAnim:SetHide(false);
-        Controls.TutorialSlowTurnEnableAnim:SetToBeginning();
-        Controls.TutorialSlowTurnEnableAnim:RegisterEndCallback(
-            function()
-                Controls.TutorialSlowTurnEnableAnim:SetHide(true);
-            end
-        );
-        Controls.TutorialSlowTurnEnableAnim:Play();
-    end
-    ContextPtr:RequestRefresh();
+    -- If the player is dead (observing), then automatically end their turn
```

Comment:
unmodified code took everything that was in this function previously and shoved it into the else case

I was going to write it like this, but I decided I wanted it to match the unmodified wherever i could

```lua
if (not pPlayerConfig:IsAlive()) then
    DoEndTurn();
    return;
end

-- Standard disable is set to false in the refresh.
-- (rest of function)
```